packages/documentation/copy/en/reference/Type Compatibility.md

```diff
@@ -36,33 +36,39 @@ TypeScript's type system allows certain operations that can't be known at compil
 
 ## Starting out
 
-The basic rule for TypeScript's structural type system is that `x` is compatible with `y` if `y` has at least the same members as `x`. For example:
+The basic rule for TypeScript's structural type system is that `x` is compatible with `y` if `y` has at least the same members as `x`. For example consider the following code involving an interface named `Pet` which has a `name` property:
 
 ```ts
-interface Named {
+interface Pet {
   name: string;
 }
 
-let x: Named;
-// y's inferred type is { name: string; location: string; }
-let y = { name: "Alice", location: "Seattle" };
-x = y;
+let pet: Pet;
+// y's inferred type is { name: string; owner: string; }
+let dog = { name: "Lassie", owner: "Rudd Weatherwax" };
+pet = deg;
 ```
 
-To check whether `y` can be assigned to `x`, the compiler checks each property of `x` to find a corresponding compatible property in `y`.
-In this case, `y` must have a member called `name` that is a string. It does, so the assignment is allowed.
+To check whether `dog` can be assigned to `pet`, the compiler checks each property of `pet` to find a corresponding compatible property in `dog`.
+In this case, `dog` must have a member called `name` that is a string. It does, so the assignment is allowed.
 
 The same rule for assignment is used when checking function call arguments:
 
 ```ts
-function greet(n: Named) {
-  console.log("Hello, " + n.name);
+interface Pet {
```

Comment:
I also got confused when the y was defined in a code block above, and though repeating that definition here makes it easier to read the example in isolation without skipping back over the previous paragraph.